docs/usage.md

```diff
@@ -83,29 +214,292 @@ nextflow pull nf-core/oncoanalyser
 
 It is a good idea to specify a pipeline version when running the pipeline on your data. This ensures that a specific version of the pipeline code and software are used when you run your pipeline. If you keep using the same tag, you'll be running the same version of the pipeline, even if there have been changes to the code since.
 
-First, go to the [nf-core/oncoanalyser releases page](https://github.com/nf-core/oncoanalyser/releases) and find the latest version number - numeric only (eg. `1.3.1`). Then specify this when running the pipeline with `-r` (one hyphen) - eg. `-r 1.3.1`.
+First, go to the [nf-core/oncoanalyser releases page](https://github.com/nf-core/oncoanalyser/releases) and find the latest pipeline version - numeric only (eg. `1.3.1`). Then specify this when running the pipeline with `-r` (one hyphen) - eg. `-r 1.3.1`. Of course, you can switch to another version by changing the number after the `-r` flag.
+
+This version number will be logged in reports when you run the pipeline, so that you'll know what you used when you look back in the future. For example, at the bottom of the MultiQC reports.
+
+To further assist in reproducbility, you can share and re-use [parameter files](#running-the-pipeline) to repeat pipeline runs with the same settings without having to write out a command with every single parameter.
+
+:::tip
+If you wish to share such profile (such as upload as supplementary material for academic publications), make sure to NOT include cluster specific paths to files, nor institutional specific profiles.
+:::
+
+## Advanced usage
+
+### Selecting processes
+
+Most of the major components in oncoanalyser can be skipped using `--processes_exclude` (the full list of available
+processes can be view [here](https://github.com/nf-core/oncoanalyser/blob/1.0.0/lib/Constants.groovy#L36-L56)).
+Multiple processes can be given as a comma-separated list. While there are some use-cases for this feature (e.g.
+skipping resource intensive processes such as VIRUSBreakend), it becomes more powerful when combined with existing
+inputs as described in the following section.
+
+:::warning
+
+When skipping components no checks are done to identify orphan processes in the execution DAG or for redundant
+processes.
+
+:::
+
+### Existing inputs
+
+The oncoanalyser pipeline has been designed to allow entry at arbitrary points, which is particularly useful in
+situations where previous outputs exist and re-running oncoanalyser is desired (e.g. to subsequently execute an
+optional sensor or use an upgrade component such as PURPLE). The primary advantage of this approach is that only the
```

Comment:
What do you mean by "optional sensor" or "upgrade component"?

Reply:
Fixed grammar and clarified.

An optional sensor would be a tool not run by default, and an upgraded component would be where new tool is released and made available in oncoanalyser.